toolkit/scripts/srpm_pack.mk

```diff
@@ -39,6 +39,10 @@ $(call create_folder,$(BUILD_DIR))
 $(call create_folder,$(BUILD_SRPMS_DIR))
 $(call create_folder,$(SRPM_BUILD_CHROOT_DIR))
 
+# Host's extended ACLs influence the default permissions of the
+# files inside the built RPMs. Disabling them for the build directory.
+$(call shell_real_build_only, setfacl -bn $(BUILD_DIR))
```

Comment:
do you think any of the other directories would need the same fix?

Reply:
The chroots where we build the packages are inside `$(BUILD_DIR)`, so this should be enough and a [PR check pipeline test](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=399535&view=results) has finally succeeded with this tweak. If we find more issues later, we can always update this change.

Reply:
Do we want this in toolkit.mk or utils.mk? It's not really unique to srpm_packing, and we will run the `$(call shell_real_build_only,...` functions before we build regardless of which `*.mk` file they are in (they are run at parse time, not execution time).

Reply:
I added the call next to where `BUILD_DIR` is created to minimize any build issues, but I don't like `sprm_pack.mk` as a place for that as well. I'm not sure why it even creates `BUILD_DIR`. I agree that `tools.mk` is better.